java/code/src/com/redhat/rhn/domain/server/ServerFactory.java

```diff
@@ -693,7 +693,7 @@ public static boolean isPtfUninstallationSupported(Server server) {
             }
 
             final RpmVersionComparator rpmVersionComparator = new RpmVersionComparator();
-            if (server.isSLES15()) {
+            if (server.isSLES15() || server.isSLES16()) {
```

Comment:
I think we do not need a version check for SLES16. All zypp versions of SLES16 are ok.
Can we directly return "true"